akka-docs/src/main/paradox/common/cluster.md

```diff
@@ -1,20 +1,17 @@
 # Cluster Specification
 
-@@@ note
+This document describes the design concepts of Akka Cluster. For the guide on using Akka Cluster please see either
+* @ref:[Cluster Usage](../typed/cluster.md)
+* @ref:[Cluster Usage with classic Akka APIs](../cluster-usage.md)
```

Comment:
this bullet list is not rendered correctly, need blank line?

Reply:
Fixed in the move.